.travis.yml

```diff
@@ -2,7 +2,7 @@ language: go
 go_import_path: github.com/kubeflow/mpi-operator
 
 go:
-- 1.11.4
+- 1.13
```

Comment:
Also update the `Dockerfile` to use 1.13?

Reply:
Updated. Although I am still trying to figure out why Travis is failing.